src/constants.js

```diff
@@ -212,19 +212,23 @@ Constants.Routes = {
   },
   UPLOAD_COURSE_REGISTRATION: {
     path: '/technical-qualification/upload-course-registration',
-    pageHeading: 'Upload the course registration email or letter'
+    pageHeading: 'Upload the course registration email or letter',
+    checksHeading: 'Course registration:'
```

Comment:
I'd prefer these to be called 'checkYourAnswersHeading' rather than 'checksHeading'

Reply:
Please note that I noticed that the checksHeading properties are now no longer used after a last minute change in the design yesterday so I have removed them altogether.